oozie-to-airflow/tests/converter/test_oozie_parser.py

```diff
@@ -405,8 +405,9 @@ class TestOozieExamples(unittest.TestCase):
             (
                 WorkflowTestCase(
                     name="decision",
-                    node_names={"fake_end", "fail", "decision_node"},
+                    node_names={"real_end", "fake_end", "fail", "decision_node"},
```

Comment:
In general I'm not a fan of the current example workflow XML with the misleading `real_end` and `fake_end`, so I wouldn't replicate it here in the test. Also it might be a good time to change the example XML.

What I'd suggest doing looks like this (the end result in Airflow):
![IMG_20190418_142137__01](https://user-images.githubusercontent.com/4280682/56360686-ac70f600-61e5-11e9-81c9-74fcceb560b4.jpg)

This way we could clearly show which path continues, and which ends the workflow.

Reply:
I will introduce a new workflow in a separate PR. Thanks for the suggestions.

Reply:
Can you link here this PR or at least the issue for this? Otherwise I'm afraid this can be forgotten.

Reply:
I created a issues: https://github.com/GoogleCloudPlatform/cloud-composer/issues/117